xml/http_servers.xml

```diff
@@ -1151,18 +1152,29 @@
   <fingerprint pattern="^Sun GlassFish Enterprise Server v(\S+)$">
     <description>Glassfish with version information</description>
     <example service.version="2.1">Sun GlassFish Enterprise Server v2.1</example>
-    <param pos="0" name="service.vendor" value="Sun"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
```

Comment:
All CPEs issued for Glassfish use `oracle` as the vendor:
https://nvd.nist.gov/products/cpe/search/results?keyword=glassfish&status=FINAL&orderBy=CPEURI&namingFormat=2.3